### PR TITLE
More comprehensive docs for e2e function tests

### PR DESCRIPTION
**Pull request by attwad**
Covers retrieval, parsing and UI actions that need to be taken to test locally.

### CODE REVIEW

**Comment by attwad** [comment]
> Great, thanks for adding more info. Just one thing - not sure I have access to the /sources page as all I get is a blank page - do we have to be given the rights to that? If so, might be useful to add that in the README file

Did you follow the instructions to make you an admin? does http://localhost:3002/profile mentions "admin" somewhere?
Please check https://github.com/globaldothealth/list/blob/main/dev/README.md#permissions

**Comment by AnyaLindstromBattle** [comment]
Ah I see it's the same steps as in https://github.com/globaldothealth/list/blob/main/dev/README.md#permissions - sorry about that

**Comment by AnyaLindstromBattle** [comment]
Having said that, it still doesn't work. Do I need a specific <YOUR-GOOGLE-LOGIN>?

**Comment by attwad** [comment]
> Having said that, it still doesn't work. Do I need a specific YOUR-GOOGLE-LOGIN?

Please connect to slack, we'll debug together there ;)

File: ingestion/functions/README.md [hunk 1]
```diff
@@ -94,27 +94,53 @@ more generally about Python Lambda development
...
+
+Go to the UI at http://localhost:3002/sources and add a new source for your parser, once you give it a name, a URL and save it, it will be given an ID.
+
+Put that ID in the `retrieval/valid_scheduled_event.json` file.
+
+Next invoke the `RetrievalFunction` like this:
 
 ```shell
 sam build
+SOURCE_API_URL=http://localhost:3001/api sam local invoke "RetrievalFunction" -e retrieval/valid_scheduled_event.json --docker-network=host
```
Comment (axmb) [inline ingestion/functions/README.md:120]: This is the value of `SOURCE_API_URL` in `template.yaml` -- is it somehow overridden, and thus we need to specify it here?

File: ingestion/functions/README.md [hunk 2]
```diff
@@ -94,27 +94,53 @@ more generally about Python Lambda development
...
+
+In your parser package's `input_event.json` set the `s3Key` as `5f311a9795e338003016593a/2020/08/10/1009/content.csv` and the `sourceId` to `5f311a9795e338003016593a`. **Set values according to the output you got, do not copy-paste what's written in this document.**
+
+Next you can invoke your parsing function:
+
+```shell
 sam local invoke "MyFunction" -e my/dir/input_event.json --docker-network=host
 ```
 
+
```
Comment (axmb) [inline ingestion/functions/README.md:136]: nit: extra ws

File: ingestion/functions/README.md (hunk 1 above)
Comment (attwad) [reply ingestion/functions/README.md:120]: ah yeah no, no need for that sorry, removed.

File: ingestion/functions/README.md (hunk 2 above)
Comment (attwad) [reply ingestion/functions/README.md:136]: hate ws

**Pull request merged**